backend/tests/integration/alembic/test_alembic.py

```diff
@@ -9,172 +9,189 @@
 
     assert "ix_records_metadata_json_referenced_records_2_0" in _get_indexes(
-        "records_metadata", db
+        "records_metadata", database
     )
 
     alembic.upgrade(target="b646d3592dd5")
 
-    assert "idx_citations_cited" in _get_indexes("record_citations", db)
+    assert "idx_citations_cited" in _get_indexes("record_citations", database)
 
-    assert "record_citations" in _get_table_names(db)
+    assert "record_citations" in _get_table_names(database)
 
     alembic.upgrade(target="5ce9ef759ace")
 
     assert "ix_legacy_records_mirror_last_updated" in _get_indexes(
-        "legacy_records_mirror", db
+        "legacy_records_mirror", database
     )
     assert "ix_legacy_records_mirror_valid_collection" in _get_indexes(
-        "legacy_records_mirror", db
+        "legacy_records_mirror", database
     )
-    assert "legacy_records_mirror" in _get_table_names(db)
+    assert "legacy_records_mirror" in _get_table_names(database)
 
     alembic.upgrade(target="c6570e49b7b2")
 
-    assert "records_citations" in _get_table_names(db)
-    assert "record_citations" not in _get_table_names(db)
+    assert "records_citations" in _get_table_names(database)
+    assert "record_citations" not in _get_table_names(database)
 
-    assert "ix_records_citations_cited_id" in _get_indexes("records_citations", db)
-    assert "idx_citations_cited" not in _get_indexes("records_citations", db)
+    assert "ix_records_citations_cited_id" in _get_indexes(
+        "records_citations", database
+    )
+    assert "idx_citations_cited" not in _get_indexes("records_citations", database)
 
     alembic.upgrade(target="dc1ae5abe9d6")
 
-    assert "idx_pid_provider" in _get_indexes("pidstore_pid", db)
+    assert "idx_pid_provider" in _get_indexes("pidstore_pid", database)
 
     alembic.upgrade(target="788a3a61a635")
 
-    assert "idx_pid_provider" not in _get_indexes("pidstore_pid", db)
+    assert "idx_pid_provider" not in _get_indexes("pidstore_pid", database)
 
     alembic.upgrade(target="e5e43ad8f861")
-    assert "ix_files_object_key_head" in _get_indexes("files_object", db)
+    assert "ix_files_object_key_head" in _get_indexes("files_object", database)
 
     alembic.upgrade(target="f563233434cd")
 
-    assert "conference_literature" in _get_table_names(db)
+    assert "conference_literature" in _get_table_names(database)
     assert "ix_conference_literature_literature_uuid" in _get_indexes(
-        "conference_literature", db
+        "conference_literature", database
     )
     assert "ix_conference_literature_conference_uuid" in _get_indexes(
-        "conference_literature", db
+        "conference_literature", database
+    )
+    assert "enum_conference_to_literature_relationship_type" in _get_custom_enums(
+        database
     )
-    assert "enum_conference_to_literature_relationship_type" in _get_custom_enums(db)
 
 
-def _get_indexes(tablename, db):
+def _get_indexes(tablename, db_alembic):
```

Comment:
db_alembic?

Reply:
there is no db_alembic anymore, I found a way to run it on normal fixture like other tests.

Reply:
you are using ``db_alembic`` :P

Reply:
no I'm not ;)